### PR TITLE
LPS-150888  [CP2.0] Automated test Add

### PR DESCRIPTION
**Pull request by styller1**
https://issues.liferay.com/browse/LPS-150888

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Review by AfonsoHSantos — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase [hunk 1]
```diff
@@ -96,7 +96,7 @@ definition {
 			CP2NavigationMenu.clickOnProductButton(product = "${testProduct}");
 		}
 
-		task ("Open the dropdown and verify that there are 5 DXP versions") {
+		task ("Open the dropdown and assert if have 5 versions") {
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase:99]: I think the "Open the dropdown and verify that there are 5 DXP versions" is better written than the current one. Would you please return it to "Open the dropdown and verify that there are 5 DXP versions" @styller1 ?

Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase:99]: Alright

**Comment by barrosdennys** [comment]
ci:forward

**Pull request closed without merging**